guides/dai_transaction_from_browser.md

```diff
@@ -0,0 +1,65 @@
+# Transacting DAI on OmiseGO Network through the browser
+
+Estimated time: 5 minutes
+
+Goal:
+
+- To understand plasma transactions with ERC20 token using DAI
+- To get first hand experience in making ERC20 interactions with plasma
+- To learn the process of making `deposit`, `transact` and `exit` with ERC20
+
+In this guide, we will introduce to you the Plasma Interface by making transactions directly from your browser. Just make sure you have the following installed:
+1. node-js >10.0.0
+2. npm > 6.0.0
+
+Have access to a code editor, browser with a web3 wallet like MetaMask
+
+### Note
+
+This guide is meant for **Samrong** and any v0.2 instance of OMG Network
+
+### Initial Setup
+1. Clone the latest version of `js-starter-kit` onto your local machine. The repo can be found here https://github.com/omisego/js-starter-kit
+
+2. Open up the `config.js` on your text editor
+
+3. Replace the current configs in `config.js` with the  [following Network Endpoints for Samrong](https://github.com/omisego/dev-portal/blob/master/guides/network_endpoints.md) then save the file.
+
+4. install all the dependencies `npm install`
+
+5. start the app up by running `npm run dev`
+
+#### 1. Funding the wallet with DAI
+
+1.1. the clientside wallet will rely on the funds you have on your existing account. In order to run the JS-Starter-kit, make sure you have sent yourself some Rinkeby Testnet DAI. We will use that later to make ERC20 transactions. DAI ERC20 contract can be found here: 
+
+https://rinkeby.etherscan.io/address/0xb307901ac0a807402a99879a491836697fec5e62
+
```

Comment:
### Fund your wallet
The client-side wallet relies on the funds you have on your existing account. 

Before you can start transacting on the OmiseGO Network, you'll need to fund your wallet with some Rinkeby Testnet DAI. 

You'll need DAI for the following: 
* To run the JS-Starter-kit
* To make ERC20 transactions

You can find DAI ERC20 contract here:
 https://rinkeby.etherscan.io/address/0xb307901ac0a807402a99879a491836697fec5e62